specs-go/config.go

```diff
@@ -406,6 +408,58 @@ type Anet struct {
 	Macaddress string `json:"macAddress,omitempty"`
 }
 
+// Windows defines the runtime configuration for Windows based containers, including Hyper-V containers.
+type Windows struct {
+	// Resources contains information for handling resource constraints for the container.
+	Resources *WindowsResources `json:"resources,omitempty"`
+}
```

Comment:
You don't seem to define this in the Markdown.  If you really want:

```json
{
  …
  "windows": {
    "resources": {
      "memory": {…}
    }
  }
}
```

instead of:

```json
{
  …
  "windows": {
    "memory": {…}
  }
}
```

You should drop a `resources` note in the Markdown.